Tests/Docker/JobDefinitionParserTest.php

```diff
@@ -425,4 +426,305 @@ public function testSimpleConfigDataWithConfigId()
         $this->assertFalse($parser->getJobDefinitions()[0]->isDisabled());
         $this->assertEmpty($parser->getJobDefinitions()[0]->getState());
     }
+
+    public function testMultiRowConfigurationWithValidProcessors()
```

Comment:
Jakto, že je to platný config? Vidím procesor v konfiguraci a vidím i procesor v rows. Na tom by měla failnout [ta podmínka `$hasProcessors && $hasRowProcessors`](https://github.com/keboola/docker-bundle/pull/305/files#diff-ae3c8131fcc10d267daffdfab1ec692fR63), ne?

Reply:
Aha, oops, nepřečetl jsem si komentář :) Sorry

Reply:
https://github.com/keboola/docker-bundle/pull/305#issuecomment-389622985